src/k8s/cmd/k8s/k8s_config.go

```diff
@@ -7,16 +7,22 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var (
+	configCmdOpts struct {
+		server string
+	}
+)
+
 func newKubeConfigCmd() *cobra.Command {
-	return &cobra.Command{
-		Use:     "config",
+	configCmd := &cobra.Command{
+		Use:     "config --server <server>",
```

Comment:
you don't need to flags here - cobra will take care of this automatically. `config` is sufficient.